docs/components/Navigation.js

```diff
@@ -60,6 +61,17 @@ export default function Navigation(): Node {
           maxHeight="calc(100% - 60px)"
           minWidth={MIN_NAV_WIDTH_PX}
         >
+          <SidebarCategorizationButton
```

Comment:
@ayeshakmaz how do we now switch the sidebar view on smaller width screeens?

![image](https://user-images.githubusercontent.com/127199/156202624-86a913e8-b09e-4066-8ac0-790290af1849.png)

Reply:
Added the toggles to the mobile view, need to double check with Hector about placement